### PR TITLE
Updates to single fetch docs

### PR DESCRIPTION
**Pull request by brophdawg11**

### CODE REVIEW

File: docs/guides/single-fetch.md [hunk 1]
```diff
@@ -119,7 +157,9 @@ export default function Component() {
...
+With Single Fetch, if an `action` returns or throws a `Response` with a `4xx/5xx` status code, Remix will _not revalidate_ loaders by default. If an `action` returns or throws anything that is not a 4xx/5xx Response, then the revalidation behavior is unchanged. The reasoning here is that in most cases, if you return a `4xx`/`5xx` Response, you didn't actually mutate any data so there is no need to reload data.
+
+If you _Want_ to continue revalidating one or more loaders after a 4xx/5xx action response, you can opt-into revalidation on a per-route basis by returning `true` from your [`shouldRevalidate`][should-revalidate] function. There is also a new `unstable_actionStatus` parameter passed to the function that you can use if you need to decode based on the action status code.
```
Comment (alexanderson1993) [inline docs/guides/single-fetch.md:162]: Just checking to make sure you want to capitalize `_Want_` here.

**Review by alexanderson1993 — APPROVED** [review]
Perfect.

File: docs/guides/single-fetch.md (hunk 1 above)
Comment (brophdawg11) [reply docs/guides/single-fetch.md:162]:
```suggestion
If you _want_ to continue revalidating one or more loaders after a 4xx/5xx action response, you can opt-into revalidation on a per-route basis by returning `true` from your [`shouldRevalidate`][should-revalidate] function. There is also a new `unstable_actionStatus` parameter passed to the function that you can use if you need to decode based on the action status code.
```

File: docs/guides/single-fetch.md [hunk 2]
```diff
@@ -119,7 +157,9 @@ export default function Component() {
...
+With Single Fetch, if an `action` returns or throws a `Response` with a `4xx/5xx` status code, Remix will _not revalidate_ loaders by default. If an `action` returns or throws anything that is not a 4xx/5xx Response, then the revalidation behavior is unchanged. The reasoning here is that in most cases, if you return a `4xx`/`5xx` Response, you didn't actually mutate any data so there is no need to reload data.
+
+If you _want_ to continue revalidating one or more loaders after a 4xx/5xx action response, you can opt-into revalidation on a per-route basis by returning `true` from your [`shouldRevalidate`][should-revalidate] function. There is also a new `unstable_actionStatus` parameter passed to the function that you can use if you need to decode based on the action status code.
```
Comment (brophdawg11) [inline docs/guides/single-fetch.md:162]:
```suggestion
If you _want_ to continue revalidating one or more loaders after a 4xx/5xx action response, you can opt-into revalidation on a per-route basis by returning `true` from your [`shouldRevalidate`][should-revalidate] function. There is also a new `unstable_actionStatus` parameter passed to the function that you can use if you need to decide based on the action status code.
```

**Pull request merged**